src/tests/networking_tests.rs

```diff
@@ -2761,6 +2857,7 @@ pub mod net_tests {
     }
 
     #[test]
+    #[ignore]
```

Comment:
whats the current behaviour?

Reply:
yeah this one definitely shouldnt fail so we need to look into it

Reply:
some mistake on my end, removed the ignore statement